geoips/schema/product_templates/sectored_xarray_dict_to_output_format.yaml

```diff
@@ -0,0 +1,23 @@
+type: object
+required:
+  - interface
+  - metadata
+  - spec
+properties:
+  interface:
+    type: string
+  metadata:
+    type: object
+    properties:
+      name:
+        type: string
+      family:
+        type: string
+      short-description:
+        type: string
+      long-description:
+        type: string
+  spec:
+    type: object
+    additionalProperties: true  # family specifies order of ops, but not YAML format
```

Comment:
The rest of the families have "additionalProperties: true" since there would otherwise be NOTHING in the spec at all...

I think if we keep these with additionalProperties: true, we should rename them all to "unvalidated_*" to be clear that they are non-standard un-validated families (basically used for development and testing, until you determine the appropriate family, or create a new one)